proto/steward/v4/cellar_v2.proto

```diff
@@ -231,6 +248,19 @@ message CellarV2_2 {
             SetStrategistPayoutAddress set_strategist_payout_address = 7;
             // Represents function `swapPositions(uint256 index1, uint256 index2)`
             SwapPositions swap_positions = 8;
+
```

Comment:
Here I believe we are also missing

`SetRebalanceDeviation`
`SetShareLockPeriod`
`InitiateShutdown`
`LiftShutdown`
`SetStrategistPlatformCut`